test/web3/walletClient.spec.ts

```diff
@@ -15,12 +19,12 @@ const receiverPrivateKey =
   'S1eK3SEXGDAWN6pZhdr4Q7WJv6UHss55EB14hPy4XqBpiktfPu6';
 
 // for CI testing:
-// const publicApi = 'https://test.massa.net/api/v2:33035';
-// const privateApi = 'https://test.massa.net/api/v2:33034';
+const publicApi = 'https://mock-public-api.com';
+const privateApi = 'https://mock-private-api.com';
 
-// For local testing:
-const publicApi = 'http://127.0.0.1:33035';
-const privateApi = 'http://127.0.0.1:33034';
+// // For local testing:
+// const publicApi = 'http://127.0.0.1:33035';
+// const privateApi = 'http://127.0.0.1:33034';
```

Comment:
Can you please use the same technic as in the last PR that needed to have both local and CI configuration?

Reply:
It's not necessary anymore, I'm mocking every call to the node. So these urls don't matter, i'll change it to mock url and remove the local testing url when every PR will be merged